apps/finance/contracts/Finance.sol

```diff
@@ -312,7 +311,7 @@ contract Finance is EtherTokenConstant, IsContract, AragonApp {
     */
     function executePayment(uint256 _paymentId)
         external
-        authP(EXECUTE_PAYMENTS_ROLE, arr(_paymentId, payments[_paymentId].amount))
+        authP(EXECUTE_PAYMENTS_ROLE, arr(_paymentId, recurringPayments[_paymentId].amount))
         paymentExists(_paymentId)
```

Comment:
Let's rename this `recurringPaymentExists` to align it with the name of the mapping.